anvil-appcompat-v7/src/gen-appcompatV7-1.1.0/kotlin/dev/inkremental/dsl/androidx/appcompat/view/menu/ActionMenuItemView.kt

```diff
@@ -17,7 +17,7 @@ import kotlin.CharSequence
 import kotlin.Suppress
 import kotlin.Unit
 
-fun actionMenuItemView(configure: ActionMenuItemViewScope.() -> Unit = {}) =
+fun ActionMenuItemView(configure: ActionMenuItemViewScope.() -> Unit = {}) =
```

Comment:
What are the reasons that we are changing dsl calls to uppercase?

Reply:
Seems to be an error, thanks for catching.